src/scripts/index.js

```diff
@@ -1,7 +1,14 @@
 // Include your code here
 import './styles/main.scss';
 import displayPokemons from './modules/generators.js';
+import displayPopup from './modules/popup.js';
+import XButton from '../assets/icons/close.svg';
 
 window.onload = async () => {
+  const close = document.querySelector('.popup-close');
+  const xImg = document.createElement('img');
+  xImg.src = XButton;
+  close.appendChild(xImg);
```

Comment:
I think it would be better to include this within the ```displayPokemons()``` function to enhance readability. Since that element is only required within that function, it can be defined inside. What do you think?

Reply:
Done!